### PR TITLE
Changed version of babel from 6 to 7

### LINKED ISSUE #174: Make sure all of the build stuff is on Babel 7

### PR DESCRIPTION
**Pull request by vikas199**
Changing the version of babel  6 to babel 7

Resolves #174

### CODE REVIEW

**Comment by NagaPradeepGajula-osi** [comment]
@vikas199 lets link the actual issue here in this PR so we would have a track of it and can close it automatically

**Comment by vikas199** [comment]
done

**Comment by NagaPradeepGajula-osi** [comment]
Awesome thanks

**Comment by eckdanny-osi** [comment]
@vikas199 see https://help.github.com/articles/closing-issues-using-keywords/

```diff
- resolves the issue: #174
+ resolves #174
```

**Comment by eckdanny-osi** [comment]
[preview deploy](https://deploy-preview-47--cws-cares.netlify.com/) looks 👍

File: package.json [hunk 1]
```diff
@@ -42,19 +42,21 @@
...
     "@storybook/react": "^3.4.11",
-    "babel-cli": "^6.26.0",
+    "babel-core": "^7.0.0-bridge.0",
     "babel-eslint": "10.0.1",
-    "babel-jest": "^23.6.0",
-    "babel-plugin-external-helpers": "^6.22.0",
-    "babel-plugin-transform-class-properties": "^6.24.1",
-    "babel-plugin-transform-object-rest-spread": "^6.26.0",
-    "babel-plugin-transform-rename-import": "^2.3.0",
+    "babel-jest": "^23.4.2",
```
Comment (eckdanny-osi) [inline package.json]: u want to _downgrade_ `babel-jest`?

Comment (vikas199) [reply package.json]: it is better to use latest version as we are using babel/core of version 7

**Comment by eckdanny-osi** [comment]
78abc47122fe19a784f5e0e42d90c4d21f3cb489 fixes the test suite:

```
yarn test --projects lib/*

Test Suites: 2 skipped, 40 passed, 40 of 42 total
Tests:       3 skipped, 202 passed, 205 total
Snapshots:   0 total
Time:        5.528s, estimated 6s
Ran all test suites in 6 projects.
✨  Done in 6.50s.
```

**Comment by eckdanny-osi** [comment]
@vikas199 do you **not** get [the results (above)](https://github.com/ca-cwds/design-system/pull/369#issuecomment-456206063) when running tests?

File: package.json (hunk 1 above)
Comment (eckdanny-osi) [reply package.json]: 🤔💭

```diff
- "babel-jest": "^23.6.0",
+ "babel-jest": "^23.4.2",
```
`23.6.0` _is_ more recent and _is_ (at time of writing this) npm `@latest` tag.

**Comment by eckdanny-osi** [comment]
i think this has sat sufficiently long. merging...

**Pull request merged**